packages/vital-meta/doc/ShadowingMeta.md

```diff
@@ -2,38 +2,44 @@
 
 Define a Shadowing token collection as defined in [Shadow](../VitalElements/Shadow).
 
-File to shadow: `/packages/PACKAGENAME/src/shadow/vital-meta/MetaHelmet.js`
+File to shadow: `/packages/{my-package}/src/shadow/vital-meta/MetaHelmet.js`
```

Comment:
@hvanyo, I used `{my-package}` just to be consistent with what I did with the Shadowing updates in the Vital List PR.

**Question:** Should `@bodiless` be in this path — i.e., `/packages/{my-package}/src/shadow/@bodiless/vital-meta/MetaHelmet.js`?
* If not, I'll want to remove it from the paths I updated in the Vital List PR.

Reply:
✅  Fixed!